cobra/core/model.py

```diff
@@ -729,7 +733,16 @@ def constraints(self):
     def exchanges(self):
         """Exchange reactions in model.
 
-        Reactions that either don't have products or substrates.
+        Reactions that exchange mass with the exterior. Uses annotations
+        and heuristics to exclude non-exchanges such as sink reactions.
+        """
+        return exchanges(self, ext_compartment=None)
```

Comment:
I would like to add corresponding properties for demands and sinks.

Reply:
As single one for both? Or a separate one for each?

Reply:
A separate one for each.